tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -1607,6 +1609,25 @@ export const showHiddenResources = async (page: Page): Promise<void> => {
   await page.locator(filesViewOptionButton).click()
 }
 
+export const toggleFlatList = async (page: Page): Promise<void> => {
+  await page.locator(filesViewOptionButton).click()
+  await page.locator(flatListToggleButton).click()
+  // close the files view option
+  await page.locator(filesViewOptionButton).click()
+}
+
+export const getAllFiles = async (page: Page): Promise<string[]> => {
+  await page.locator(footerTextSelector).isVisible()
```

Comment:
use `waitFor()` if you are checking for element visibility. `isVisible` will return true or false only.
```suggestion
  await page.locator(footerTextSelector).waitFor()
```